config.mk

```diff
@@ -38,3 +38,10 @@ else
 	SHLIB_SUFFIX    :=.so
 endif
 endif
+
+SBOX=.cabal-sandbox
+ifneq "$(wildcard $(SBOX))" ""
+	IDRIS :=.cabal-sandbox/bin/idris
```

Comment:
Does this cause the libraries to be built with the just-built Idris, or with the one that was last installed in the sandbox?